schema.json

```diff
@@ -2097,11 +2097,23 @@
         "force_reload": true,
         "id": "show_powered_by"
       },
+      {  
+        "type": "checkbox",
+        "label": "Hide Brands in Footer",
+        "force_reload": true,  
+        "id": "shop_by_brand_hide_footer"
+      },
       {
         "type": "checkbox",
         "label": "Show Copyright Info",
         "force_reload": true,
         "id": "show_copyright_footer"
+      },
+      {  
+        "type": "checkbox",
+        "label": "Hide Brands in Footer",
+        "force_reload": true,  
+        "id": "shop_by_brand_hide"

```

Comment:
Is this setting used anywhere?
I only see `shop_by_brand_hide_footer` being used in the footer template